tests/components/hvv_departures/test_config_flow.py

```diff
@@ -0,0 +1,258 @@
+"""Test the HVV Departures config flow."""
+from asynctest import patch
+from pygti.exceptions import CannotConnect, InvalidAuth
+
+from homeassistant.components.hvv_departures import config_flow
+from homeassistant.components.hvv_departures.const import CONF_STATION
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+
+
+async def test_user_flow(hass):
+    """Test that config flow works."""
+    flow = config_flow.ConfigFlow()
```

Comment:
Please start the tests with the core interfaces. Use `hass.config_entries.flow.async_init`. I recommend using our scaffold script to generate example code with our current best practices.